index.html

```diff
@@ -9,20 +9,24 @@
 
     <section></section>
     </head>
+    <section><img src="photos/photoNicole.jpeg" alt="Nicole" width="300" height="300" style="border-radius: 50%"></section>
     <section>
-        <h1>ABOUT:</h1>
-        <h2>Hi, my name is Nicole Bradley. I love computer coding. It is my dream to one day be a game developer. I am playing a game called Star Trek Fleet Command. I have to play the games I want to learn how to make. </h2>
+        <h1 style="color:purple;">ABOUT:</h1>
+        <h2>Hi, my name is Nicole Bradley. I love computer coding. It is my dream to one day be a game developer. 
+            I am playing a game called Star Trek Fleet Command. 
+            I have to play the games I want to learn how to make. </h2>
```

Comment:
I would just use `<p>` for this: `<h*>` should only be used for section headings.